scripts/init-countries.js

```diff
@@ -102,17 +134,25 @@ async function main () {
             .then(res => {
                 console.log(res)
             })
-            .catch(err => console.log(err))
-            .then(process.exit(0))
+            .catch(err => { 
+                console.error(err);
+                console.error("--- Can't insert cities, exiting ---")
+                process.exit(20)
+            })
+            .then(_ => {
+                console.log('--- Countries and cities inserted successfully ---');
+                process.exit(0);
+            })
 
     } catch (error) {
-        console.log(error)
-        process.exit(0)
+        console.error(error)
+        console.error("--- Failed to init countries and cities, exiting ---")
+        process.exit(2)
     }
 }
 
 async function getCountries () {
-    return axios.get('https://restcountries.com/v3/all')
+    return axios.get('https://raw.githubusercontent.com/threefoldtech/tfchain_graphql/master/scripts/countries.json')
```

Comment:
why do we need to fetch the data remotely while we have it locally?

Reply:
So we don't need a release or/and a new image in case we want to modifiy/update these data.